contracts/utils/CloneFactory.sol

```diff
@@ -0,0 +1,55 @@
+pragma solidity ^0.4.24;
+
+/*
+The MIT License (MIT)
+
+Copyright (c) 2018 Murray Software, LLC.
+
+Permission is hereby granted, free of charge, to any person obtaining
+a copy of this software and associated documentation files (the
+"Software"), to deal in the Software without restriction, including
+without limitation the rights to use, copy, modify, merge, publish,
+distribute, sublicense, and/or sell copies of the Software, and to
+permit persons to whom the Software is furnished to do so, subject to
+the following conditions:
+
+The above copyright notice and this permission notice shall be included
+in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY
+CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+*/
+
+/**
+ * @title EIP 1167: Minimal Proxy Contract
+ *
+ * @dev To simply and cheaply clone contract functionality in an immutable way,
+ * this standard specifies a minimal bytecode implementation that delegates
+ * all calls to a known, fixed address.
+ *
+ * https://eips.ethereum.org/EIPS/eip-1167
+ */
+contract CloneFactory {
+
+  event CloneCreated(address indexed target, address clone);
+
+  function createClone(address target) internal returns (address result) {
+    // solium-disable-next-line
+    bytes memory clone = hex"3d602d80600a3d3981f3363d3d373d3d3d363d73bebebebebebebebebebebebebebebebebebebebe5af43d82803e903d91602b57fd5bf3";
```

Comment:
@nventuro Nah, I'm not sure why it differs, my low level staff knowledge is still weak :/ I hope you could help me with understanding. 

For this I just took Peter's  [CloneFactory](https://github.com/optionality/clone-factory/blob/master/contracts/CloneFactory.sol). The mentioned bytecode listed in the EIP comes from [ContractProbe](https://github.com/optionality/clone-factory/blob/master/contracts/ContractProbe.sol).

I need to take some lessons for `assembly` staff because I've feeling that it's the reason :)

Reply:
Hah, it's good you have concerns here :) It's time to learn instructional assembly and figure out what does the extra bytecode do :)

Reply:
You can use https://solmap.zeppelin.solutions/ to debug that - just paste the assembly code on the left tab!

Reply:
@nventuro I'm coming with some updates that sti
* This extra 10 bytes at the beginning are just initial code for the contract. It's described already [here](https://github.com/optionality/clone-factory/blob/43da836da9016a18fa08bbcfcf7655e1a95683bf/clone-contract.js#L36) - It stands for copying final contract code into memory. 
* It was skipped in essential code introduced in the [EIP](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1167.md#specification)

Reply:
I see, that's what I thought. Thanks!